src/styles/components/_quote-box.scss

```
@@ -0,0 +1,99 @@
+@import '../abstracts/_abstracts-dir';
+
+.quote-box {
+  width: 100%;
+  height: fit-content;
+
+  @include respond-above('md') {
+    margin-top: rem(24px);
+  }
+
+  &__container {
+    margin: rem(16px);
+
+    @include respond-above('md') {
+      margin: rem(24px) 0 0 rem(24px);
+    }
+
+    @include respond-above('lg') {
+      margin-left: rem(32px);
+    }
+
+    &--background-image {
+      background: {
+        image: url('/images/double-quote-left.png'),
+          url('/images/double-quote-left.png');
+        repeat: no-repeat;
+        size: rem(14px 12px), rem(14px 12px);
+        position: top left, bottom right;
+      }
+
+      @include respond-above('sm') {
+        background-size: rem(22px 20px), rem(22px 20px);
+      }
+
+      @include respond-above('md') {
+        background-size: rem(26px 24px), rem(26px 24px);
+      }
+
+      @include respond-above('lg') {
+        background-size: rem(30px 28px), rem(30px 28px);
+      }
+
+      @include respond-above('xl') {
+        background-size: rem(36px 32px), rem(36px 32px);
```

Actually, I tried `background-size: rem(x y)` and it applied for both components. 
```suggestion
        size: rem(14px 12px);
        position: top left, bottom right;
      }

      @include respond-above('sm') {
        background-size: rem(22px 20px);
      }

      @include respond-above('md') {
        background-size: rem(26px 24px);
      }

      @include respond-above('lg') {
        background-size: rem(30px 28px);
      }

      @include respond-above('xl') {
        background-size: rem(36px 32px);
```

---

Good catch!